<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuAjouter</class>
 <widget class="QWidget" name="MenuAjouter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QFrame#frameGen{
background: white;
}



#frame{
border: 3px solid gray;
background: gray
}



QGroupBox {
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QPushButton#ajouterProduit {
color: white;
background-color: rgb(63, 124, 146);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 90px;
max-width: 90px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
max-width: 150px;
min-width: 100px;
}

QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


 
QPushButton#ajouterIngredient {
color: white;
background-color: rgb(63, 124, 146);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 90px;
max-width: 90px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
max-width: 150px;
min-width: 100px;
}

QPushButton#ajouterRecette {
color: white;
background-color: rgb(63, 124, 146);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 90px;
max-width: 90px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
max-width: 150px;
min-width: 100px;
}

QPushButton#retour {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 90px;
max-width: 90px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
max-width: 150px;
min-width: 100px;
}</string>
  </property>
  <widget class="QFrame" name="frameGen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>471</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="ajouterProduit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>148</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Produit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouterIngredient">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>94</y>
        <width>148</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Ingredient</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouterRecette">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>138</y>
        <width>148</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Recette</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Menu Ajouter Produits &amp; Recettes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retour">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>retour</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
